src/main/scala/li/cil/oc/server/component/traits/InventoryTransfer.scala

```diff
@@ -48,7 +48,7 @@ trait InventoryTransfer extends traits.WorldAware with traits.SideRestricted {
     val sinkSide = checkSideForAction(args, 1)
     val sinkPos = position.offset(sinkSide)
     val count = args.optFluidCount(2)
-    val sourceTank = args.optInteger(3, -1)
+    val sourceTank = args.optInteger(3, -1) - 1
```

Comment:
@repo-alt What do you think?

Reply:
It will break some code of course, and there is no easy way to notify the user about that change, but the change makes sense indeed, so I don't think you should look back at us.

Reply:
Understood. In this case I'll probably keep it for a larger release, like 1.9.0 (if it ever happens - at this rate, I doubt it!).